pkg/controller/hawtio/hawtio_controller.go

```diff
@@ -535,7 +577,24 @@ func (r *ReconcileHawtio) Reconcile(request reconcile.Request) (reconcile.Result
 			}
 		}
 	}
+	// Reconcile the client certificate cronjob
+	cronJob := &v1beta1.CronJob{}
+	cronJobName := request.Name + "-certificate-expiry-check"
 
+	if cronJobErr := r.client.Get(ctx, client.ObjectKey{Namespace: request.Namespace, Name: cronJobName}, cronJob); cronJobErr == nil {
+		if hawtio.Spec.Auth.ClientCertCheckSchedule != "" {
+			if cronJob.Spec.Schedule != hawtio.Spec.Auth.ClientCertCheckSchedule {
+				cronJob.Spec.Schedule = hawtio.Spec.Auth.ClientCertCheckSchedule
```

Comment:
Should the expiration period be checked also?

Reply:
I added the period check too in the last update.